api/coreservice.go

```diff
@@ -145,8 +145,6 @@ type (
 		PendingNonce(address.Address) (uint64, error)
 		// ReceiveBlock broadcasts the block to api subscribers
 		ReceiveBlock(blk *block.Block) error
-		// BlockHashByActionHash returns block hash by action hash
-		BlockHashByActionHash(h hash.Hash256) (hash.Hash256, error)
```

Comment:
run `make mockgen` to update mock service file

Reply:
done